proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -392,3 +399,24 @@ enum Infraction {
 message ValidatorUpdates {
   repeated tendermint.abci.ValidatorUpdate updates = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
+
+// ConsPubKeyRotationHistory contains a validator's consensus public key rotation history.
+message ConsPubKeyRotationHistory {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_getters)  = false;
+
+  // operator_address defines the address of the validator's operator; bech encoded in JSON.
+  string operator_address = 1 [(gogoproto.moretags) = "yaml:\"operator_address\""];
+  // old_cons_pubkey is the old consensus public key of the validator, as a Protobuf Any.
+  google.protobuf.Any old_cons_pubkey = 2 [
+    (cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey",
+    (gogoproto.moretags)             = "yaml:\"old_consensus_pubkey\""
```

Comment:
Is it still something we do, YAML annotations?

Reply:
I guess these are not needed, thanks for pointing @julienrbrt